jetty-ee10/jetty-ee10-webapp/src/main/java/org/eclipse/jetty/ee10/webapp/MetaInfConfiguration.java

```diff
@@ -160,8 +162,8 @@ public void findAndFilterContainerPaths(final WebAppContext context) throws Exce
         if (classPath != null)
         {
             Stream.of(classPath.split(File.pathSeparator))
-                .map(URIUtil::toURI)
-                .filter(uriPatternPredicate)
+                .map(resourceFactory::newResource)
+                .filter(r -> uriPatternPredicate.test(r.getURI()))
```

Comment:
You're dereferencing a potentially null resource.

Reply:
only in exceptional cases that probably should throw rather than return null.. but OK...